presto-main/src/main/java/com/facebook/presto/util/DateTimeUtils.java

```diff
@@ -70,6 +70,8 @@ public static String printDate(int days)
         return DATE_FORMATTER.print(TimeUnit.DAYS.toMillis(days));
     }
 
+    @Deprecated
```

Comment:
Don't mark as deprecated. This applies to all `@Deprecated` in this pull request.

I kind of see why you have `@Deprecated` after reading this rest of this pull request. You are using that to mark a method as legacy. You avoided renaming by using method overload for the new methods. To compensate for the confusion this could lead to, you then use `@Deprecated` to mark the legacy one. People reading the code at the callsites will have to rely on strike-through lines in the IDE to distinguish between the two.

I see that all the deprecated ones takes a tz, while new ones don't. This is a little bit subtle in my taste.

An alternative is to rename the old method through with two steps:
* commit 1 renames `xyz` to `legacyXyz`.
* commit 2 changes `legacyXyz` to use `LEGACY_TIMESTAMP_WITHOUT_TIME_ZONE_FORMATTER`, and adds back `xyz` with new semantics.

I prefer renaming. What do you say @martint?